src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -1928,9 +1967,9 @@ class VariantSampleMatrix[T](val hc: HailContext, val metadata: VariantMetadata,
 
   /**
     *
-    * @param config VEP configuration file
-    * @param root Variant annotation path to store VEP output
-    * @param csq Annotates with the VCF CSQ field as a string, rather than the full nested struct schema
+    * @param config    VEP configuration file
+    * @param root      Variant annotation path to store VEP output
+    * @param csq       Annotates with the VCF CSQ field as a string, rather than the full nested struct schema
```

Comment:
I switched to the Hail sytle_guide, but these aligned indentations persisted after auto-formatting and auto-indenting. Should I manually undo them? They look good.